backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/CsvProcessingException.java

```diff
@@ -5,8 +5,11 @@
 import graphql.GraphQLError;
 import graphql.language.SourceLocation;
 import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
 
 /** An error thrown when CSV uploads fail for reasons other than validation */
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Since allows us to set the response status to 400 when we throw this error inside the upload controller